crates/analytics/docs/clickhouse/scripts/merchant_account.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE merchant_account_queue (
+    `merchant_id` String,
+    `publishable_key` String,
+    `sign_flag` Int8
+) ENGINE = Kafka SETTINGS kafka_broker_list = 'kafka0:29092',
+kafka_topic_list = 'hyperswitch-merchant-account',
+kafka_group_name = 'hyper',
+kafka_format = 'JSONEachRow',
+kafka_handle_error_mode = 'stream';
+
+CREATE TABLE  merchant_account(
+    `merchant_id` String,
+    `publishable_key` String,
+    `sign_flag` Int8,
+    INDEX publishableKeyIndex publishable_key TYPE bloom_filter GRANULARITY 1,
+) ENGINE = CollapsingMergeTree(sign_flag) PARTITION BY toStartOfDay(created_at)
+ORDER BY
+    (publishable_key) TTL toStartOfDay(created_at) + toIntervalMonth(18) SETTINGS index_granularity = 8192;
+
+CREATE MATERIALIZED VIEW merchant_account_mv TO merchant_account (
+    `merchant_id` String,
+    `publishable_key` String,
+    `sign_flag` Int8,
```

Comment:
merchant_id and publishable_key are immutable, ie, no need to go with a CMT

Reply:
Never mind bro this PR is not needed only, as no event related to merchant_account is pushed to kafka. Instead we manually created a table in clickhouse prod.